Jobs.Fetcher.YouTube/YouTubeFetchers.cs

```diff
@@ -109,12 +109,14 @@ public class YouTubeFetchers : FetcherJobsFactory {
             }
 
             return new List<AbstractJob>() {
-                       new VideosQuery(youtubeServices),
-                       new PlaylistsQuery(youtubeServices),
                        new DailyVideoMetricsQuery(youtubeServices),
-                       new ViewerPercentageMetricsQuery(youtubeServices),
-                       new StatisticsQuery(youtubeServices),
+                       new PlaylistsQuery(youtubeServices),
```

Comment:
Not really important. I thought it could make sense to have it sorted alphabetically. Also would be nice to do this in imports and other lists where order is not relevant